src/discord/staff/censor.py

```diff
@@ -145,5 +143,11 @@ async def censor_remove(
                 )
 
 
+def censorify_word(phrase_to_bleep: str) -> str:
```

Comment:
I don't think there's much of a reason for this to be a global. Please add this into the class.